### PR TITLE
feat(vm-logs): settings

### PR DESCRIPTION
**Pull request by wowshakhov**

### CODE REVIEW

File: src/app/settings/settings.module.ts [hunk 1]
```diff
@@ -1,10 +1,8 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { ClipboardModule, ClipboardService } from 'ngx-clipboard';
-
```
Comment (tamazlykar) [inline src/app/settings/settings.module.ts]: It's a good idea to separate third party imports and local imports. This described in Angular Style Guide

File: src/app/settings/settings.module.ts [hunk 2]
```diff
@@ -15,6 +13,10 @@ import {
   ThemeSelectorComponent,
   VmSettingsComponent,
 } from './components';
+import { UpdateButtonFieldComponent } from './components/update-button-field/update-button-field.component';
+import { VmLogsMessagesComponent } from './components/vm-logs-settings/vm-logs-messages/vm-logs-messages.component';
```
Comment (tamazlykar) [inline src/app/settings/settings.module.ts]: Why you did not add components in index.ts and did not export them as above?

File: src/app/vm-logs/redux/vm-logs.effects.ts [hunk 3]
```diff
@@ -106,12 +107,13 @@ export class VmLogsEffects {
   @Effect()
   enableAutoUpdate$: Observable<Action> = this.actions$.pipe(
     ofType(vmLogsActions.VmLogsActionTypes.ENABLE_AUTO_UPDATE),
-    switchMap(() =>
-      timer(0, 10000).pipe(
+    withLatestFrom(this.store.pipe(select(UserTagsSelectors.getVmLogsShowLastMinutes))),
+    switchMap(([_, minutes]) =>
```
Comment (tamazlykar) [inline src/app/vm-logs/redux/vm-logs.effects.ts]: Use implicit name instead of `_`

File: src/app/settings/settings.module.ts (hunk 1 above)
Comment (wowshakhov) [reply src/app/settings/settings.module.ts]: Add optimize imports sort by module

**Pull request merged**